parser-typechecker/src/Unison/Codebase/BranchUtil.hs

```diff
@@ -69,10 +70,10 @@ getBranch (p, seg) b = case Path.toList p of
     (Branch.head <$> Map.lookup h (b ^. Branch.children))
       >>= getBranch (Path.fromList p, seg)
 
-makeAddTermName :: Path.Split -> Referent -> (Path, Branch0 m -> Branch0 m)
```

Comment:
I generalized these splits because some of the update combinators now take Absolute splits (which is clearer anyways, since it's not ambiguous what the path is relative to)